google-cloud-bigtable/src/main/java/com/google/cloud/bigtable/data/v2/models/RowMutation.java

```diff
@@ -49,11 +63,33 @@ public static RowMutation create(@Nonnull String tableId, @Nonnull String key) {
     return create(tableId, ByteString.copyFromUtf8(key));
   }
 
+  /**
+   * Creates a new instance of the mutation builder on an authorized view.
+   *
+   * <p>See {@link com.google.cloud.bigtable.admin.v2.models.AuthorizedView} for more details about
+   * an AuthorizedView.
+   */
+  public static RowMutation createForAuthorizedView(
+      @Nonnull String tableId, @Nonnull String authorizedViewId, @Nonnull String key) {
```

Comment:
`Precondition.checkNotNull(authorizedViewId);`

The `@Nonnull` annotation is for code mantainers to know the expected object value. it doesn't really enforce anything.